sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/Book.java

```diff
@@ -0,0 +1,83 @@
+package io.quarkus.ts.reactive;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import io.quarkus.hibernate.reactive.panache.PanacheEntityBase;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@Entity
+@Table(name = "books")
+@NamedQuery(name = "title_prefix", query = "from Book where title like :prefix")
+public class Book extends PanacheEntityBase {
```

Comment:
what about this one?

Reply:
As I understand it, Quarkus generates getters and setters, which can be used by Hibernate, but cannot be used by client code. Since we need to use these getters and setters in client code, we have to define them manually.

Reply:
Agree